udata/models/__init__.py

```
@@ -180,8 +180,10 @@ def init_app(app):
     for plugin in app.config['PLUGINS']:
         name = 'udata.ext.{0}.models'.format(plugin)
         try:
-            __import__(name)
-        except ImportError:
-            pass
-        except Exception as e:
+            module_obj = importlib.import_module(name)
+            for item in getattr(module_obj, '__all__'):
+                globals()[item] = getattr(module_obj, item)
+        except ImportError as e:
```

That silent error cost me one afternoon, I really think we should keep it, logs are cheap :)